chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -578,7 +578,7 @@ async def test_same_sb_twice_with_eligible_coin() -> None:
     sb = SpendBundle.aggregate([sb1, sb2])
     sb_name = sb.name()
     result = await add_spendbundle(mempool_manager, sb, sb_name)
-    expected_cost = uint64(10268283)
+    expected_cost = uint64(6600088)
```

Comment:
It's an opportunity to make these numbers easier to read.
```suggestion
    expected_cost = uint64(6_600_088)
```